dependencies.gradle

```diff
@@ -44,15 +44,13 @@ ext {
     navigationVersionName = "1.0.0-alpha1"
     // ----------------------------------
 
-    //Config Android sdk, plugins and libraries version
+    // Config Android sdk, plugins and libraries version
     gradlePluginVersion = "7.0.2"
     kotlinVersion = "1.5.0"
     dokkaVersion = "1.6.0"
-    compileSdkVersion = 29
```

Comment:
If possible, we should keep the ScreenManager components with the old compileSdkVersion and targetSdkVersion so they are in sync with the artifacts

Reply:
Fixed